code/game/machinery/iv_drip.dm

```diff
@@ -145,6 +145,8 @@
 				update_icon()
 
 /obj/machinery/iv_drip/attack_hand(mob/user)
+	if (!ishuman(user))
```

Comment:
Well, I based it on the logic for attaching IVs, and that is, borgs can't. That means there's a lot more of an issue here than just IV detachment.